index.js

```diff
@@ -759,3 +762,22 @@ is.hex = function (value) {
 is.symbol = function (value) {
   return typeof Symbol === 'function' && toStr.call(value) === '[object Symbol]' && typeof symbolValueOf.call(value) === 'symbol';
 };
+
+is.not = negateMethods(is);
+
+function negateMethods(source) {
+  var negatedMethods = {};
+  forEach(source, function (value, key) {
+    var negatedMethod;
+    if (isCallable(value)) {
+      negatedMethod = negatedMethods[key] = function () {
+        return !value.apply(this, arguments);
```

Comment:
I put `null` there as you wanted. To be honest, I've never understood the general phobia with `this`. It never cause me any problems, and I use it heavily. But, this is your module, so you get to make the rules hereabouts.